packages/typespec-ts/test/modularUnit/scenarios/samples/client/parameterizedHost.md

```diff
@@ -0,0 +1,117 @@
+# skip: Should generate samples for parameterized host client
```

Comment:
Skipped due to TCGC issue tracked in: https://github.com/Azure/typespec-azure/issues/1419